backend/FwLite/LcmCrdt/CrdtMiniLcmApi.cs

```diff
@@ -154,85 +150,37 @@ public async Task<ComplexFormType> CreateComplexFormType(MiniLcm.Models.ComplexF
         if (sortWs is null)
             throw new NullReferenceException($"sort writing system {options.Order.WritingSystem} not found");
         queryable = queryable
+            .LoadWith(e => e.Senses).ThenLoad(s => s.ExampleSentences)
```

Comment:
This is the big win right here isn't it? 🎉

Reply:
Yeah this is awesome